cpmpy/solvers/TEMPLATE.py

```diff
@@ -134,26 +149,22 @@ def solver_var(self, cpm_var):
         """
         # TODO: add `solver_vars(self, cpm_vars)` to SolverInterface class
 
-        if is_num(cpm_var):
```

Comment:
Won't the removal of this check mean that we must do a lot more checks in the _post_constraint? Or we assume that all of the burden there has been overcome by the transformations?